content/github/getting-started-with-github/getting-started-with-git/caching-your-github-credentials-in-git.md

```diff
@@ -94,6 +94,7 @@ Turn on the credential helper so that Git will save your password in memory for
   $ git config --global credential.helper 'cache --timeout=3600'
   # Set the cache to timeout after 1 hour (setting is in seconds)
   ```
+For more options for storing your credentials on Linux, see [Section 7.14 Git Tools - Credential Storage](https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage) in Git Pro by Scott Chacon and Ben Straub.
```

Comment:
Adding this newline should prevent this sentence from being indented like this and therefore make it easier to notice! 😊 

![image](https://user-images.githubusercontent.com/14935376/119886493-a5565f00-bef8-11eb-8a07-2ac7a642d1cb.png)

```suggestion

For more options for storing your credentials on Linux, see [Credential Storage](https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage) in Pro Git.
```